modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Answer.es.js

```diff
@@ -40,13 +40,20 @@ export default ({answer}) => {
 
 	return (
 		<>
-			<div className="autofit-row autofit-padded">
+			<div
+				className={
+					'autofit-row autofit-padded ' +
+					(showAsAnswer ? 'question-accepted-answer' : '')
```

Comment:
use classname pkg

Reply:
using it here: https://github.com/boton/liferay-portal/pull/123/commits/6d26053ac69fd9bf12fd8b36baacd57acc37d03f#diff-939015cc9d7fcc9b96d84597478f1086